src/app/projects/projects.component.html

```diff
@@ -1 +1,65 @@
-<h1>PROJECTS PAGE</h1>
```

Comment:
Why is this file here ?
Aren't you creating new brach from stable master?

Reply:
@ancyp Currently the Project page has not been merged into the main gsoc18-code branch. That is why perhaps when I made a new branch from it, this default page showed up. :)